manifests/plugin/dynflow/params.pp

```diff
@@ -2,4 +2,5 @@
 class foreman_proxy::plugin::dynflow::params {
   $enabled           = true
   $listen_on         = 'https'
+  $database_path     = '/var/spool/foreman-proxy/dynflow/dynflow.sqlite'
```

Comment:
Yea, the lack of it existing in the packages is what made we realize it wasn't in the installer, will take care of that too.

I would guess it should be persistent since we could otherwise use the in-memory sqlite.

@iNecas Ok to move this to /var/lib?